neps/runtime.py

```diff
@@ -623,6 +619,9 @@ def run(self) -> None:  # noqa: C901, PLR0912, PLR0915
                     report=report,
                     worker_id=self.worker_id,
                 )
+                # This is mostly for `tblogger`
+                for _key, callback in _TRIAL_END_CALLBACKS.items():
+                    callback(trial_to_eval)
```

Comment:
what does this do here or why is it needed?
mainly to ensure its affect on the vanilla runtime